packages/gestalt/src/Table.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import * as React from 'react';
+import styles from './Table.css';
+
+const TableBody = ({ children }: { children: React.Node }) => (
+  <tbody>{children}</tbody>
+);
+
+const TableCell = ({
+  children,
+  colSpan,
+  rowSpan,
+}: {
+  children: React.Node,
+  colSpan?: number,
+  rowSpan?: number,
+}) => {
+  let props = { className: styles.td };
+  if (colSpan) {
+    props = { ...props, colSpan };
+  }
+  if (rowSpan) {
+    props = { ...props, rowSpan };
+  }
+
+  return <td {...props}>{children}</td>;
+};
+
+const TableHeader = ({ children }: { children: React.Node }) => (
```

Comment:
@PayalModi Even though these components will be quite small, can we put them in their own files? Makes it easier to expand / test there individual components.